client/src/components/pages/home/extra-work.tsx

```diff
@@ -1,13 +1,18 @@
+import ExtraWorkCard from "./extra-work-card";
 import "./extra-work.module.css";
+import { extraWorkData } from "@/data/extra-work-data";
 
 const ExtraWork = () => {
-  return (
-    <div className="extra-work">
-      <div className="w-full">
-        <h1>Extra Work</h1>
-      </div>
-    </div>
-  );
+	return (
+		<div className="extra-work p-[32px] overflow-x-scroll">
+			<h2 className="text-2xl font-bold">Check out more of our work!</h2>
```

Comment:
We'll want to use our [translation infrastructure](https://github.com/codeforboston/boston-liquor-license-tracker/tree/main/client/src/i18n) for this text.

You can see an example in [This PR for the 404 page](https://github.com/codeforboston/boston-liquor-license-tracker/pull/104/files)